pvlib/pvsystem.py

```diff
@@ -1887,6 +1887,82 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
+def pvsyst_celltemp(
+    poa_global,
+    wind_speed,
+    temp_air,
+    eta_m=0.1,
+    alpha_absorption=0.9,
+    temp_model="freestanding",
+):
```

Comment:
No other pvlib functions follow this indentation pattern. I guess it's ok according to PEP8, but consistency within the library is nice too.

Reply:
I understand and will collapse the variables so they fit in less lines. 

If you're interested, here's a rationalization for this pattern implemented by the formatter I used called [Black](https://github.com/ambv/black#how-black-wraps-lines).